components/gallery/GalleryItemAction/GalleryItemActionSection.vue

```diff
@@ -43,4 +49,46 @@ watchEffect(async () => {
 })
 </script>
 
-<style scoped></style>
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.gallery-item-action-section {
+  justify-content: space-between;
+  align-items: center;
```

Comment:
`is-justify-content-space-between is-align-items-center` could work here

Reply:
No, because I have to override the flex layout in custom CSS below for mobile and pc.